chore: Class#method refinements (#297)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/im/guide/senior.mdx

</MultiLang>

> > Java, Android SDKs offer `LCIMClient#getChatRoomQuery` methods that are dedicated to querying chat rooms. By using it, you won't need to deal with the `transient` attribute of conversations.

Reviewer: ```suggestion
> The Java (Android) SDK offers the `LCIMClient#getChatRoomQuery` method that is dedicated to querying chat rooms. By using this method, you won't need to deal with the `transient` attribute of conversations.
```
